iiop-openjdk/src/main/resources/schema/jboss-as-iiop-openjdk_1_1.xsd

```diff
@@ -22,11 +22,11 @@
   -->
 
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           targetNamespace="urn:jboss:domain:iiop-openjdk:1.0"
-           xmlns="urn:jboss:domain:iiop-openjdk:1.0"
+           targetNamespace="urn:jboss:domain:iiop-openjdk:1.1"
+           xmlns="urn:jboss:domain:iiop-openjdk:1.1"
            elementFormDefault="qualified"
            attributeFormDefault="unqualified"
-           version="1.0">
+           version="1.1">
```

Comment:
version here should stay 1.0 as that is version of xmlschema format not the subsystem.
also you renamed the file instead of adding new one.
1.0 schema should stay as part of docs.

btw, wildflx-iiop-openjdk_1_1.xsd would be better name in the context